src/main.c

```diff
@@ -47,8 +47,7 @@ static float user_configuration(bool run)
 {
 	float force;
 
-	//force = speed_mode_configuration();
-	force = 0.25;
+	force = speed_configuration();
```

Comment:
Update the header of the function (speed by force)

Reply:
ab8a77f748424eb39d0371cba65ff190b91e4f1d